cluster/src/test/scala/io/snappydata/benchmark/snappy/TPCH_Snappy_Query.scala

```diff
@@ -1,123 +1,141 @@
+            println("---------------------------------------------------------------------------------")
+        }
+
+      //      TPCH_Snappy.execute("q1", snc, isResultCollection, isSnappy, i)
+      //      TPCH_Snappy.execute("q2", snc,isResultCollection, isSnappy, i)//taking hours to execute in Snappy
+      //      TPCH_Snappy.execute("q3", snc, isResultCollection, isSnappy, i)
+      //      TPCH_Snappy.execute("q4", snc, isResultCollection, isSnappy, i)
+      //      TPCH_Snappy.execute("q5", snc, isResultCollection, isSnappy, i)
+      //      TPCH_Snappy.execute("q6", snc, isResultCollection, isSnappy, i)
+      //      TPCH_Snappy.execute("q7", snc, isResultCollection, isSnappy, i)
+      //      TPCH_Snappy.execute("q8", snc, isResultCollection, isSnappy, i)
+      //      TPCH_Snappy.execute("q9", snc, isResultCollection, isSnappy, i) //taking hours to execute in Snappy
+      //      TPCH_Snappy.execute("q10", snc, isResultCollection, isSnappy, i)
+      //      TPCH_Snappy.execute("q11", snc, isResultCollection, isSnappy, i)
+      //      TPCH_Snappy.execute("q12", snc, isResultCollection, isSnappy, i)
+      //      TPCH_Snappy.execute("q13", snc, isResultCollection, isSnappy, i)
+      //      TPCH_Snappy.execute("q14", snc, isResultCollection, isSnappy, i)
+      //      TPCH_Snappy.execute("q15", snc, isResultCollection, isSnappy, i)
+      //      TPCH_Snappy.execute("q16", snc, isResultCollection, isSnappy, i)
+      //      TPCH_Snappy.execute("q17", snc, isResultCollection, isSnappy, i)
+      //      TPCH_Snappy.execute("q18", snc, isResultCollection, isSnappy, i)
+      //      TPCH_Snappy.execute("q19", snc,isResultCollection, isSnappy, i) //not working in local mode hence not executing it for cluster mode too
+      //      TPCH_Snappy.execute("q20", snc, isResultCollection, isSnappy, i)
+      //      TPCH_Snappy.execute("q21", snc,isResultCollection, isSnappy, i) //not working in local mode hence not executing it for cluster mode too
+      //      TPCH_Snappy.execute("q22", snc, isResultCollection, isSnappy, i)
+      //      println("---------------------------------------------------------------------------------")
+    }
+
+    TPCH_Snappy.close()
+  }
+
+  override def isValidJob(sc: SnappyContext, config: Config): SnappyJobValidation = {
+
+    val sqlSparkProps = if (config.hasPath("sparkSqlProps")) {
+      config.getString("sparkSqlProps")
+    }
+    else " "
+
+    sqlSparkProperties = sqlSparkProps.split(" ")
+
+    queryPlan = if (config.hasPath("queryPlan")) {
+      config.getBoolean("queryPlan")
+    } else false
+
+    val tempqueries = if (config.hasPath("queries")) {
+      config.getString("queries")
+    } else {
+      return new SnappyJobInvalid("Specify Query number to be executed")
+    }
+
+    println(s"tempqueries : $tempqueries")
+
+    queries = tempqueries.split(",")
+
+    SnappyJobValid()
+  }
+}
```

Comment:
Check with Kishor about these large scale formatting changes. He may have many uncommitted local changes that will all conflict with above. I think it will be better to let him correct the formatting.